promoting version 6.0.20-12 (#178)
RedisLabs/redis-enterprise-k8s-docs: README.md

```
![Alt text](./images/web_console_1.png?raw=true)
2. List the secrets via the command:
```bash
    kubectl get secret
```
![Alt text](./images/web_console_2.png?raw=true)
3. Run the command below to view the secret with the credentials , replace the <cluster name> with the name of your Redis Enterprise Cluster:
```bash
    kubectl get secret <cluster name> -o yaml
```
![Alt text](./images/web_console_3.png?raw=true)
4. Extract the username and password via the commands below, replace the <cluster name> with the name of your Redis Enterprise Cluster:
```bash
    kubectl get secret <cluster name> -o jsonpath='{.data.username}' | base64 --decode
    kubectl get secret <cluster name> -o jsonpath='{.data.password}' | base64 --decode
```
![Alt text](./images/web_console_4.png?raw=true)

In order to connect to the web console please use one of the following methods:

Method 1: using port-forward
1. Get the port of the cluster UI service via the command below, replace the <cluster name> with the name of your Redis Enterprise Cluster:
```bash
    kubectl get service/<cluster name>-ui -o yaml
```
Note: the default port is 8443.
![Alt text](./images/web_console_5.png?raw=true)

> 2. Run the command below to set port-forward, replace the <cluster name> with the name of your Redis Enterprise Cluster, use the port of the service for the service port and the port you want to use on the local machine as the local port:

Reviewer: ```suggestion
2. Run the kubectl port-forward service to set port-forward. Replace the <cluster name> with the name of your Redis Enterprise Cluster, replace <service port> with the port of the service, and replace <local port> with the port you want to use on the local machine.
```

```bash
    kubectl port-forward service/<cluster name>-ui <local port>:<service port>
```
![Alt text](./images/web_console_6.png?raw=true)

> 3. In the web browser on the local machine to see the Redis Enterprise web console go to:

Reviewer: ```suggestion
3. View the web console from the web browser on your local machine: 
```

https://localhost:<local port>
Don't forget to replace the <local port> with the one used in the previous command.
![Alt text](./images/web_console_7.png?raw=true)

Method 2: load balancer

> > Note - configuring a load balancer service for the UI will create an external IP address, widely available (when set on cloud providers which support external load balancers). Use with caution. 

Reviewer: ```suggestion
<note> Configuring a load balancer service for the UI will create an external IP address, widely available (when set on cloud providers which support external load balancers). Use with caution. </note>
```
